test/integration/features/cmd-config.feature

```diff
@@ -0,0 +1,242 @@
+@cmd-config @command
+Feature: Minishift config subcommands
+Commands `minishift config [sub-command]` are used for storing
+user defined options which changes default behaviour of Minishift.
+
+  @minishift-only
+  Scenario: Config is empty
+    Given Minishift has state "Does Not Exist"
+     Then executing "minishift config view" succeeds
+      And stdout should be empty
+
+  Scenario Outline: Setting values in bad range or format
```

Comment:
This short summary is the same as the sentence below... 
I am not sure what "in bad range" would mean.

Disallowed/illegal values and bad format ?